ui/packages/app/web/src/components/ProfileSelector.tsx

```diff
@@ -76,6 +76,14 @@ const wellKnownProfiles = {
     name: 'Block Contention Time Total',
     help: 'Time delayed stack traces caused by blocking on synchronization primitives.',
   },
+  fgprof_samples_count: {
+    name: 'Fgprof Samples Total',
+    help: 'CPU profile samples observed regardless of their current On/Off CPU scheduling status',
+  },
+  fgprof_time_nanoseconds: {
+    name: 'Fgprof Samples Time Total',
+    help: 'CPU profile measured regardless of their current On/Off CPU scheduling status in nanoseconds',
+  },
```

Comment:
names and help descriptions were chosen based on my understanding of https://github.com/felixge/fgprof - if something else would be preferable, please feel free to suggest

Reply:
I think these are great!